common/models/user.js

```diff
@@ -668,6 +668,45 @@ module.exports = function(User) {
       next();
     });
 
+    // Delete old sessions once email is updated
+    var oldEmail, newEmail, oldId, newId;
```

Comment:
These variables are shared among all invocations of your hook. What will happen when there are two "save" operations in progress at the same time?